src/components/TextInput/TextInputLabel/index.js

```diff
@@ -14,8 +15,9 @@ class TextInputLabel extends PureComponent {
     render() {
         return (
             <Animated.Text
-                pointerEvents="none"
                 ref={(el) => (this.label = el)}
+                pointerEvents="none"
+                accessibilityRole={CONST.ACCESSIBILITY_ROLE_TEXT}
```

Comment:
```suggestion
                accessibilityRole={CONST.ACCESSIBILITY_ROLE.TEXT}
```

Reply:
@Skalakid we missed this and need to make this change ASAP

Reply:
@situchan @roryabraham I'm not sure what the suggested change is, can you explain?

Reply:
`CONST.ACCESSIBILITY_ROLE_TEXT` is undefined. It should be `CONST.ACCESSIBILITY_ROLE.TEXT`

Reply:
@Skalakid please look with 🦅's 👁️ to find diff 😄